lib/domain/model/ongoing_call.dart

```diff
@@ -2154,7 +2162,56 @@ class OngoingCall {
             devices.firstWhereOrNull(
               (e) => e.deviceId() == track.getTrack().deviceId(),
             );
+
+        await _addNoiseSuppression(track);
+      }
+    }
+  }
+
+  /// Adds [NoiseSuppressionLevel] to [LocalMediaTrack].
+  ///
+  /// Available only on Desktop now.
+  Future<void> _addNoiseSuppression(LocalMediaTrack track) async {
+    Log.debug('_addNoiseSuppression($track)', '$runtimeType');
+
+    if (!track.isAudioProcessingAvailable()) {
+      Log.debug(
+        '❌ Audio processing not available for this track $track',
+        '$runtimeType',
+      );
+      return;
+    }
+
+    // Currently works only on Desktop
+    if (PlatformUtils.isWeb || !PlatformUtils.isDesktop) {
+      Log.debug(
+        '❌ Noise suppression is not supported on this platform',
+        '$runtimeType',
+      );
+      return;
+    }
+
+    final enabled = _noiseSuppressionEnabled;
+    if (enabled == null) {
+      Log.debug('❌ _noiseSuppressionEnabled is null', '$runtimeType');
+      return;
+    }
```

Comment:
I don't think any logs should be printed if noise suppression is disabled or are unavailable for the current platform.